### DISCUSSION TITLE
⛔ 🚀 Feature Request: Two nice feature for accordion and taxonomy filter

### DISCUSSION THREAD

**Discussion by johanporret** [elementor/elementor#23293] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

Please make it possible to call the loops in accordions
And make it possible to also change the layout of the new filter widget by taxonomy, to drop-down for example.

### Describe the solution you'd like

1. Make it possible to call the loops in accordions

2. Change the layout of the new filter widget by taxonomy

### Describe alternatives you've considered

1. It would be really nice if we could call our CPTs in an accordion dynamically

2. It would be nice if the taxonomy widget could be by drop down, or alphabet

**Comment by nicholaszein** [comment]
Hello there @johanporret!

thanks for submitting your feature request! 🙏 

### 🗃️ Please understand that while we do welcome new feature requests, we can't track multiple requests per thread. 

Please follow our guidelines for submitting a new [🚀 Feature Request](https://github.com/elementor/elementor/discussions/new?category=feature-request), and submit only ☝️ **_<ins>one request per thread</ins>_** so we can track them better and allow users to vote only for the features they want to see implemented.

Kind regards
